tests/rule_based_profiler/test_profiler_user_workflows.py

```diff
@@ -394,11 +394,104 @@ def test_bobby_profiler_user_workflow_multi_batch(
 
     expectation_suite: ExpectationSuite = profiler.profile(
         expectation_suite_name=bobby_columnar_table_multi_batch[
-            "expected_expectation_suite_name"
-        ],
+            "test_configuration_oneshot_sampling_method"
+        ]["expectation_suite_name"],
     )
 
     assert (
         expectation_suite
-        == bobby_columnar_table_multi_batch["expected_expectation_suite"]
+        == bobby_columnar_table_multi_batch[
+            "test_configuration_oneshot_sampling_method"
+        ]["expected_expectation_suite"]
+    )
+
+
+def test_bobby_profiler_user_workflow_multi_batch_row_count_range_rule_bootstrap_sampling_method(
+    bobby_columnar_table_multi_batch_context,
+    bobby_columnar_table_multi_batch,
+):
+    # Load data context
+    data_context: DataContext = bobby_columnar_table_multi_batch_context
+    # Load profiler configs & loop (run tests for each one)
+    profiler_config: str = bobby_columnar_table_multi_batch["profiler_config"]
+
+    # Instantiate Profiler
+    full_profiler_config_dict: dict = yaml.load(profiler_config)
+    rules_configs: dict = full_profiler_config_dict.get("rules")
+    variables_configs: dict = full_profiler_config_dict.get("variables")
+
+    # Extract only the "row_count_range_rule" from the configuration and set the "sampling_method" of its only
+    # "parameter_builder" (an instance of NumericMetricRangeMultiBatchParameterBuilder) to be the "bootstrap" method.
+    # The reason for this manipulation is that even though the default value of the "sampling_method" is "bootstrap",
+    # it is set to "oneshot" in the test configuration of every "parameter_builder" (for those that are instances of
+    # NumericMetricRangeMultiBatchParameterBuilder) in order for the large expectation suite fixture, created for the
+    # parametric ("oneshot") mode of the NumericMetricRangeMultiBatchParameterBuilder class to be applicable (coverage).
+    # The "Bobby" profiler use case, applied to the Taxi data, outputs 1 table expectation and 30 column expectations.
+    # The present test of the "bootstrap" sampling method uses the expectation suite containing only one expectation,
+    # because setting up the statistical assertions (and the required fixtures with the tolerances) is very laborious.
+    # In the future, additional of the "bootstrap" sampling method should be created using domain builders that generate
+    # a small number of domains (ideally, only one domain) so as to make the building of test fixtures more manageable.
+    row_count_range_rule: dict = rules_configs["row_count_range_rule"]
+    parameter_builders: dict = row_count_range_rule["parameter_builders"]
+    row_count_range_parameter: dict = parameter_builders[0]
+    row_count_range_parameter["sampling_method"] = "bootstrap"
+    rules_configs = {
+        "row_count_range_rule": row_count_range_rule,
+    }
```

Comment:
Could this all be done in one shot just to be clearer that we are modifying a sub-item of the `rules_configs` dict?
```suggestion
    rules_configs["row_count_range_rule"]["parameter_builders"][0]["sampling_method"] = "bootstrap"
```

Reply:
Can an argument be made that the verbosity here is helpful for future users of the test? Don't think it's a big difference either way

Reply:
I'm good with either - both work so this is more stylistic. 

But hopefully this clarifies my thought process:

Usually I'm a fan of verbosity, but here it feels like the update of `sampling_method` is a bit indirect. 

I.e. as written it looks like we are making a change to `row_count_range_parameter` which then is not explicitly used in `rules_configs`. It does actually update `rules_configs` because `row_count_range_parameter` was created by drilling down into the `rules_configs` and `row_count_range_parameter` contains a reference to the sub-dict within the original `rules_configs`. This just feels more like a side effect (reminds me of using pointers) than just modifying the value in the object directly with the proposed code.

Reply:
@talagluck @cdkini Both of your comments are well-taken and made me realize that this is not a good test.  I conjured up a much better test, which will appear later (hence, this PR branch will be updated -- and I will notify you once it has been updated so that you can re-review).  Thanks!